models/demos/deepseek_v3/tt/mlp/mlp_1d.py

```diff
@@ -186,6 +188,16 @@ def prefill_model_config(cls, hf_config: PretrainedConfig, mesh_device: ttnn.Dev
 
         # Construct the config
         return {
+            "input_reshard": ReshardConfig(
```

Comment:
if this op is not used in mlp and instead used in the decoder block, then it should be defined there instead of here. This breaks abstraction.